### DISCUSSION TITLE
Setup same button size for different button text

### DISCUSSION THREAD

**Discussion by aitech95** [zauberzeug/nicegui#3923] [Q&A] [answered]
### Question

I need keep some button size (for example 200 pixel x 200 pixel) for different button text. However, the width of ui buttons will changed with different button text and there's no way to set button height. Can anyone tell me the solution?

**Comment by falkoschindler** [comment]
Hi @aitech95,

Is this what you're looking for?
```py
ui.button('Click me').classes('w-[200px] h-[200px]')
ui.button('No, click me').classes('w-[200px] h-[200px]')
```
<img width="436" alt="Screenshot 2024-10-27 at 11 32 29" src="https://github.com/user-attachments/assets/885612fd-16c3-458c-9322-493dd921f2ac">

**Reply by aitech95** [reply, marked as answer]
Hi falk schindler,
It's exactly what I'm looking for!
Thank you very much for your early reply!
